modules/simple-payments/simple-payments.php

```diff
@@ -377,5 +398,173 @@ function setup_cpts() {
 		register_post_type( self::$post_type_product, $product_args );
 	}
 
+	/**
+	 * Format a price for display
+	 *
+	 * Largely taken from WordPress.com Store_Price class
+	 *
+	 * The currency array will have the shape:
+	 *   format  => string sprintf format with placeholders `%1$s`: Symbol `%2$s`: Price.
+	 *   symbol  => string Symbol string
+	 *   desc    => string Text description of currency
+	 *   decimal => int    Number of decimal places
+	 *
+	 * @param  string $the_currency The desired currency, e.g. 'USD'.
+	 * @return ?array               Currency object or null if not found.
+	 */
+	private static function get_currency( $the_currency ) {
+		$currencies = array(
+			'USD' => array(
+				'format'  => '%1$s%2$s', // 1: Symbol 2: currency value
+				'symbol'  => '$',
+				'desc'    => _x( 'United States Dollars', 'currency', 'jetpack' ),
+				'decimal' => 2,
+			),
+			'GBP' => array(
+				'format'  => '%1$s%2$s', // 1: Symbol 2: currency value
+				'symbol'  => '&#163;',
+				'desc'    => _x( 'British Pounds', 'currency', 'jetpack' ),
+				'decimal' => 2,
+			),
+			'JPY' => array(
+				'format'  => '%1$s%2$s', // 1: Symbol 2: currency value
+				'symbol'  => '&#165;',
+				'desc'    => _x( 'Japanese Yen', 'currency', 'jetpack' ),
+				'decimal' => 0,
+			),
+			'BRL' => array(
+				'format'  => '%1$s%2$s', // 1: Symbol 2: currency value
+				'symbol'  => 'R$',
+				'desc'    => _x( 'Brazilian real', 'currency', 'jetpack' ),
```

Comment:
Is it intended to have some currencies with sentence case vs some with capital case (USD for example)?

Reply:
Descriptions are now unused and removed.